lib/capistrano/tasks/slackify.cap

```diff
@@ -7,10 +7,11 @@ namespace :slack do
 
       if fetch(:slack_notify_events).include? :started
         info 'Notifying Slack of deploy starting'
-
-        execute :curl, '-X POST', '-s', '--data-urlencode',
-          Slackify::Payload.build(self, :starting),
-          fetch(:slack_url)
+        Array(fetch(:slack_channel)).each {|channel|
+          execute :curl, '-X POST', '-s', '--data-urlencode',
+            Slackify::Payload.new(self, :starting, channel),
```

Comment:
Sorry, one more thing brought up during test -  this should be calling `build` instead of `new`